msktutil.cpp

```diff
@@ -1267,6 +1302,16 @@ int main(int argc, char *argv [])
         goto error;
     }
 
+    /* delete mode will only work with admin credentials */
+    if (exec->mode == MODE_DELETE) {
+        flags->user_creds_only = true;
+    }
+
+    /* reset mode will only work with admin credentials */
```

Comment:
While this holds true in the common case, can we be reasonably sure no-one would ever want the machine account to reset its own password? Something like: _Hey, I'll go down to re-deploy myself, so please set the machine account back to a well-known password_.

Reply:
Probably irrelevant, and not worth pursuing.